tests/global_scheduler/test_dispatch_scheduler.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2024, Alibaba Group;
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+import pytest
+
+from llumnix.instance_info import InstanceLoadCalculator, InstanceInfo
+from llumnix.global_scheduler.dispatch_scheduler import DispatchScheduler
+
+
+def init_dispatch_scheduler(policy='load'):
+    instance_load_calculator = InstanceLoadCalculator('remaining_step', True)
+    dispatch_scheduler = DispatchScheduler(policy, instance_load_calculator)
+    return dispatch_scheduler
+
+@pytest.fixture
+def dispatch_scheduler():
+    dispatch_scheduler = init_dispatch_scheduler()
+    yield dispatch_scheduler
+
+def test_add_instance_and_remove_instance(dispatch_scheduler):
+    dispatch_scheduler.add_instance('instance_1')
+    assert dispatch_scheduler.num_instance == 1
+    dispatch_scheduler.add_instance('instance_2')
+    assert dispatch_scheduler.num_instance == 2
+    dispatch_scheduler.remove_instance('instance_1')
+    assert dispatch_scheduler.num_instance == 1
+    dispatch_scheduler.remove_instance('instance_2')
+    assert dispatch_scheduler.num_instance == 0
+
+def test_dispatch():
```

Comment:
just testing single dispatch policy in a function is more clearer. 
e.g. test_balanced_dispatch_policy, test_load_diapatch_policy...

in addition, add test for Flood dispatch policy

Reply:
Get it, and flood dispatch policy unittest should be added by you after you have finished your development :)

Reply:
Already resolved.